examples/gaussian.py

```diff
@@ -15,6 +15,7 @@
 model = A*exp(-((x - x0)**2/(2 * sig**2)))
 
 # Sample 10000 points from a N(15.0, 1.5) distrubution
+np.random.seed(seed=123456789)
```

Comment:
It might have improved but its hard to tell because of the probabilistic nature of fitting. It sometimes works, sometimes not, roughy 1/3 times not.